appveyor.yml

```diff
@@ -47,10 +47,10 @@ install:
       if ($env:CONDA -eq "true") {
         conda config --set always_yes yes --set changeps1 no;
         conda install -q pip setuptools wheel numpy scipy;
-        pip install -r develop-requirements.txt;
-        pip install .
+        pip install --disable-pip-version-check -r develop-requirements.txt;
```

Comment:
Rather than setting these options on each command I prefer to set an environment variable `PIP_DISABLE_PIP_VERSION_CHECK=yes`.

Reply:
In fact, I just set this variable through the interface so you don't have to worry about it. These options can simply be removed.

Reply:
Would it bother if I still set the environment variable in the config? Just so that it is explicit and people could fork the config and it would still work.

Reply:
Good idea, please do.